core/esmf-aspect-model-validator/pom.xml

```diff
@@ -43,7 +43,8 @@
       <dependency>
          <groupId>org.graalvm.js</groupId>
          <artifactId>js</artifactId>
-         <type>pom</type>
+         <type>jar</type>
+         <version>${graalvm-version}</version>
```

Comment:
Why do you need the version and type here?

Reply:
If you install type pom, plugin will exclude this dependency from the fat jar. But in documentation this is not so clear. Regarding version, to be more consistent with current version of graalvm

Reply:
But you already declared that in parent POM. IMHO should also work without version and type

Reply:
Yes, but I tried this without type and version and maven started excluding this package from fat jar...
But, let's try some new build

Reply:
I can't understand it, it also works locally for me and also with the maven plugin.
I think the problem lies somewhere else. There is no type pom for this deps. Thats the reason why you don't need type and version because you declared it in parent pom